apps/cowswap-frontend/src/common/pure/CurrencySelectButton/styled.tsx

```diff
@@ -1,4 +1,4 @@
-import { ReactComponent as DropDown } from '@cowprotocol/assets/images/dropdown.svg'
+import DropDown from '@cowprotocol/assets/images/dropdown.svg?react'
```

Comment:
Updated following https://github.com/pd4d10/vite-plugin-svgr

Reply:
Cool, I was wondering about that